docker-compose.yml

```diff
@@ -56,24 +59,51 @@ services:
       timeout: 20s
       retries: 10
 
+  mysql-read-replica:
```

Comment:
Can we also explicitly specify replicas for all the microservices with a default value of `1`?
Ref: https://docs.docker.com/compose/compose-file/compose-file-v3/#replicas

Reply:
Applied.